server/src/api/site.rs

```diff
@@ -130,6 +130,7 @@ pub struct GetSiteResponse {
   pub online: usize,
   version: String,
   my_user: Option<User_>,
+  allowed_instances: Vec<String>,
```

Comment:
I think it would be better to name this `federated_instances` or similar, so we dont have to rename this once we add blocklist based federation.

Reply:
mmk, done.